civiclens/tests/nlp_test_data/mock-row.json

```diff
@@ -0,0 +1,42 @@
+{
+  "doc_plain_english_title": "Student Loan Debt Waiver: Department Of Education",
+  "document_id": "ED-2023-OPE-0123-26398",
+  "id": "7588edfc-4239-4970-970e-d080eecf4da7",
+  "last_updated": "May 6, 2024",
+  "num_rep_comments": 2,
+  "num_topics": 2,
+  "num_total_comments": 980,
+  "num_unique_comments": 762,
+  "rep_comments": [
+    {
```

Comment:
I see rep comments is a list here with the topic added. Where is the topic section added and rep_comments converted to a list? I'm not finding it

Reply:
see the above comment on the list, and then I'll had the topics when I run it through the topic_model so that's not reflected in this pr